client/src/stories/Button.js

```diff
@@ -1,50 +1,32 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import './button.css';
-
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+import ButtonPlus from "./assets/buttonPlus.svg";
 /**
  * Primary UI component for user interaction
  */
-export const Button = ({ primary, backgroundColor, size, label, ...props }) => {
-  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
+
+const ButtonElement = styled.div`
+  border-radius: 50%;
+`;
+
+const ButtonImage = styled.img``;
+
+export const Button = ({ label }) => {
```

Comment:
I would expect that `Button` is rendering a `button` element. 
But it renders this:
```html
<div>
  <img src="/assets/buttonPlus.svg" />
  LABEL
</div>
```

If you take a look at this HTML, I would consider renaming your component (or add `button`).
The image makes the button very specific because a plus is always rendered.
Maybe name it `PlusButton`?